janusgraph-driver/src/test/java/org/janusgraph/graphdb/tinkerpop/JanusGraphIoRegistryTest.java

```diff
@@ -111,7 +116,7 @@ public void testJanusGraphPredicatesAsGryo() throws SerializationException {
     }
 
     @Test
-    public void testGeoshapAsGryo() throws SerializationException {
+    public void testGeoshapeAsGryo() throws SerializationException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/JanusGraph/janusgraph/pullRequest?prid=4880769)